NAMESPACE

```diff
@@ -9,6 +9,8 @@ export(orderly_bundle_pack)
 export(orderly_bundle_run)
 export(orderly_cleanup)
 export(orderly_commit)
+export(orderly_config)
+export(orderly_config_)
```

Comment:
I don't think we should export the class itself - I know that means documenting fairly manually at present, but I think we're not necessarily at the point where we can stand behind the API here either.

Reply:
Thanks, tidied that up now.